### PR TITLE
[DT Kit] bugfixes for R24-08

### LINKED ISSUE #973: [Digital Twin KIT] Remove API Plug-In
Please refer to https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/issues/970 to see tasks and acceptance criteria.

### PR DESCRIPTION
**Pull request by arnoweiss**
## Description

closes https://github.com/eclipse-tractusx/sig-release/issues/806
closes #973

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by arnoweiss** [comment]
I couldn't find the exact release dates for R24-05 and R24-08. Please add them via the [github-suggestion](https://stackoverflow.com/questions/54239733/how-can-i-manually-add-suggestions-in-code-reviews-on-github)
mechanism to this PR during review in the changelog.md.

**Review by jSchuetz88 — APPROVED** [review]
Hi @arnoweiss,
thank you for updating the references and especially for removing the deprecated API-files. Looks good to me.

@danielmiehle @maximilianong

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ security setup etc.). As the information contained in the DTR may be sensitive a
 every data provider must offer his own DTR as an EDC Data Asset. While it is only mandatory to implement the GET
 endpoints
 as specified in
-the [Development View](API%20AAS%20Registry/dotaas-part-2-http-rest-registry-service-specification.info.mdx),
+the [Development View](../../../../openApi/dt/registry-service-ssp002-openapi.yaml),
```
Comment (jSchuetz88) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:72]: Hi @arnoweiss, I just saw, that you updated the relative path to an absolute path for the Submodel API. Maybe that one should be changes to an absolute path as well?

**Review by danielmiehle — APPROVED** [review]
LGTM. Thank you for the changes!

**Review by maximilianong — APPROVED** [review]

**Pull request merged**